org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/workflowEngine/Message.java

```diff
@@ -0,0 +1,78 @@
+package org.eclipse.ice.modeling.workflowEngine;
+
+public class Message {
+
+	/**
+	 * DEPRECATE - This is part of the DataSet MetaData
```

Comment:
Should this be here, if it is a part of MetaData? Or is this something else that is separate from MetaData?

Reply:
There is actually a large discussion that is needed around messages, what they contain, etc.  Specifically, as I have not real had information yet, this is a place where my thinking is evolving from just having some attributes in a message to understanding that the same information may be contained in the meta data.  So there are two thing going on here.  1) this is a concept in the process of changing and 2) we need to start getting specific about the data model and what data we are actually working with.  I only have notional concepts that tend to change every meeting.